### ISSUE TITLE
Missing dependency to Tk

### ISSUE THREAD

**Issue by joefresna** [ludericv/information-market#2]
I've got this error:

```
$ python src/info_market.py config/config.txt
Traceback (most recent call last):
  File "/Users/joefresna/information-market/src/info_market.py", line 1, in <module>
    from controllers.main_controller import MainController, Configuration
  File "/Users/joefresna/information-market/src/controllers/main_controller.py", line 3, in <module>
    from src.model.environment import Environment
  File "/Users/joefresna/information-market/src/model/environment.py", line 2, in <module>
    from PIL import Image, ImageTk
  File "/Users/joefresna/information-market/infomarket-env/lib/python3.9/site-packages/PIL/ImageTk.py", line 28, in <module>
    import tkinter
  File "/usr/local/Cellar/python@3.9/3.9.6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/tkinter/__init__.py", line 37, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
ModuleNotFoundError: No module named '_tkinter'
```

**Comment by joefresna** [comment]
I'm now trying to install Tk with
```
$ brew install python-tk
```

but it's taking much longer than expected. Let you know how it goes.

**Comment by ludericv** [comment]
> I've got this error:
> 
> ```
> $ python src/info_market.py config/config.txt
> Traceback (most recent call last):
>   File "/Users/joefresna/information-market/src/info_market.py", line 1, in <module>
>     from controllers.main_controller import MainController, Configuration
>   File "/Users/joefresna/information-market/src/controllers/main_controller.py", line 3, in <module>
>     from src.model.environment import Environment
>   File "/Users/joefresna/information-market/src/model/environment.py", line 2, in <module>
>     from PIL import Image, ImageTk
>   File "/Users/joefresna/information-market/infomarket-env/lib/python3.9/site-packages/PIL/ImageTk.py", line 28, in <module>
>     import tkinter
>   File "/usr/local/Cellar/python@3.9/3.9.6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/tkinter/__init__.py", line 37, in <module>
>     import _tkinter # If this fails your Python may not be configured for Tk
> ModuleNotFoundError: No module named '_tkinter'
> ```

This is weird, from what I know tkinter is directly included with the standard python distribution. Maybe you can try using the *python3* command instead of *python* so:
```python3 info_market.py ../config/config.txt```

**Comment by ludericv** [comment]
> I've got this error:
> 
> ```
> $ python src/info_market.py config/config.txt
> Traceback (most recent call last):
>   File "/Users/joefresna/information-market/src/info_market.py", line 1, in <module>
>     from controllers.main_controller import MainController, Configuration
>   File "/Users/joefresna/information-market/src/controllers/main_controller.py", line 3, in <module>
>     from src.model.environment import Environment
>   File "/Users/joefresna/information-market/src/model/environment.py", line 2, in <module>
>     from PIL import Image, ImageTk
>   File "/Users/joefresna/information-market/infomarket-env/lib/python3.9/site-packages/PIL/ImageTk.py", line 28, in <module>
>     import tkinter
>   File "/usr/local/Cellar/python@3.9/3.9.6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/tkinter/__init__.py", line 37, in <module>
>     import _tkinter # If this fails your Python may not be configured for Tk
> ModuleNotFoundError: No module named '_tkinter'
> ```

This is weird, from what I know tkinter is directly included with the standard python distribution. Maybe you can try using the *python3* command instead of *python* so:
```python3 info_market.py ../config/config.txt```

**Comment by joefresna** [comment]
I've installed Tk and it now works! :-)

Regarding your last suggestion, I ran the command from the virtual environment that was built with python3.9, so already calling `python` used that version of python... therefore, I don't think using `python3` would have changed the issue...
Now I cannot try it anymore, as after installation it works.

Don't know... maybe we should test it on a third machine and in case Tk is needed, we can add it in the Requirements section.
